pom.xml

```diff
@@ -339,4 +394,42 @@
 
     </profiles>
 
+    <name>maven-parent</name>
+    <description>Parent pom for the Samply projects</description>
+    <url>https://github.com/samply/maven-parent</url>
```

Comment:
Ist <url> hier sinnvoll oder schlägt sich das auf alle abgeleiteten POMs durch?

Reply:
Tut es meines Erachtens nicht: https://maven.apache.org/guides/introduction/introduction-to-the-pom.html#Project_Inheritance